processing/src/test/java/org/apache/druid/query/aggregation/first/DoubleFirstAggregationTest.java

```diff
@@ -30,14 +30,16 @@
 import org.apache.druid.query.aggregation.TestObjectColumnSelector;
 import org.apache.druid.segment.ColumnSelectorFactory;
 import org.apache.druid.segment.column.ColumnHolder;
+import org.apache.druid.testing.InitializedNullHandlingTest;
 import org.easymock.EasyMock;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
 import java.nio.ByteBuffer;
+import java.util.Comparator;
 
-public class DoubleFirstAggregationTest
+public class DoubleFirstAggregationTest extends InitializedNullHandlingTest
```

Comment:
it looks like these tests will only check `useDefaultValuesForNull = true` or is there some config I'm not seeing that sets it to `false` as well in another run?

Reply:
travis runs tests in both modes, this is to let tests run in intellij which doesn't get initialized correctly